contracts/SnappAuction.sol

```diff
@@ -140,17 +209,37 @@ contract SnappAuction is SnappBase {
         uint128 buyAmount, 
         uint128 sellAmount
     ) 
-        internal pure returns (bytes32) 
+        internal view returns (bytes32) 
```

Comment:
@josojo  can you please explain the change here? I see it is conflicting with master and wonder which to choose?

Reply:
I guess the changes that have been made here are no longer "pure"

Reply:
yeah, if you use variables from inherited contracts, you can no longer use pure.